### PR TITLE
52 misc 10 update tests

### PR DESCRIPTION
**Pull request by SMateen21**
Updated test classes to now reflect the use of enums instead of Strings for card values, now the tests compile.
Not all the tests pass due to changes in main. I'd like to merge these into main and start on a new branch containing the latest changes as rebasing as a mess in intellij.

### CODE REVIEW

File: src/main/java/entities/deck_logic/StandardDeck.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
-package entities;
+package entities.deck_logic;
 
+import entities.card_logic.Card;
+import entities.deck_logic.Deck;
```
Comment (meiiph) [inline src/main/java/entities/deck_logic/StandardDeck.java:4]: Duplicate import statement

File: src/main/java/entities/game_logic/GameObserver.java [hunk 2]
```diff
@@ -1,4 +1,6 @@
-package entities;
+package entities.game_logic;
+
+import entities.game_logic.GameAccess;
```
Comment (meiiph) [inline src/main/java/entities/game_logic/GameObserver.java:3]: Duplicate import statement

File: src/main/java/entities/game_logic/ObservableGame.java [hunk 3]
```diff
@@ -1,6 +1,6 @@
-package entities;
+package entities.game_logic;
 
-import java.util.ArrayList;
+import entities.game_logic.GameObserver;
```
Comment (meiiph) [inline src/main/java/entities/game_logic/ObservableGame.java:3]: Duplicate import statement

File: src/main/java/entities/game_logic/ObserverNotifier.java [hunk 4]
```diff
@@ -1,4 +1,7 @@
-package entities;
+package entities.game_logic;
+
+import entities.game_logic.IObserverNotifier;
+import entities.game_logic.ObservableGame;
```
Comment (meiiph) [inline src/main/java/entities/game_logic/ObserverNotifier.java:4]: Duplicate import statements

File: src/main/java/entities/player_logic/Player.java [hunk 5]
```diff
@@ -1,4 +1,9 @@
-package entities;
+package entities.player_logic;
+
+import entities.card_logic.Card;
+import entities.deck_logic.Deck;
+import entities.game_logic.Game;
+import entities.player_logic.Hand;
```
Comment (meiiph) [inline src/main/java/entities/player_logic/Player.java:6]: Duplicate import statement

File: src/main/java/entities/player_logic/PlayerFactory.java [hunk 6]
```diff
@@ -1,4 +1,7 @@
-package entities;
+package entities.player_logic;
+
+import entities.player_logic.Hand;
+import entities.player_logic.Player;
```
Comment (meiiph) [inline src/main/java/entities/player_logic/PlayerFactory.java:4]: Duplicate import statements

File: src/test/java/use_cases/PlayerGameInteractorTest.java [hunk 7]
```diff
@@ -56,21 +66,50 @@ public void setUp() {
...
+        return null;
+    }
+    private boolean hasAnyValid(Player player) {
+        for(Card card: player.getCards()) {
+            if (manager.isValidCard(card)) {
+                return true;
+            }
+        }
+        return false;
+    }
```
Comment (meiiph) [inline src/test/java/use_cases/PlayerGameInteractorTest.java:112]: If we could add javadocs to the three above methods that would be good :)

**Review by meiiph — APPROVED** [review]
Looks good, just some minor details with the import statements, and some missing javadocs. None of them are code breaking, so I'll approve, but if those could get addressed whenever possible, that'd be great :)

**Review by lilyxmeng — APPROVED** [review]
Just javadocs

**Review by nyan-ko — DISMISSED** [review]
same as nancy's review

**Comment by nyan-ko** [comment]
merging with prior reviews, addressed small merge conflict

**Pull request merged**